build.gradle

```diff
@@ -69,11 +69,11 @@ dependencies {
     compile ("com.pahimar.ee3:EquivalentExchange3:${ee3_version}:dev") {
         transitive = false
     }
-    
```

Comment:
all those whitespace changes will make it a pain to merge this into the 1.9 bracnh